### DISCUSSION TITLE
Suspicious phishing email

### DISCUSSION THREAD

**Discussion by Smilkoski** [community/community#118668] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I got an email from noreply@xn--gthub-tsa.com where key was added to my account, I didn't add key to my account.
![image](https://github.com/community/community/assets/29953083/8254654b-a079-48c9-9932-7ff68dfce826)

**Comment by Akash1134** [comment]
Hi @Smilkoski ,

 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
